components/voting.tsx

```diff
@@ -472,6 +415,58 @@ export default class Voting extends React.PureComponent<VotingProps, VotingState
             <em>No sessions yet.</em>
           </p>
         )}
+        {this.state.show === 'votes' && <p>This year we're doing preferential voting.</p>}
+
+        {this.state.show === 'all' && (
```

Comment:
tag cloud looks good. Would prefer it as a component as this page is noisy but up to you